src/session.rs

```diff
@@ -51,7 +51,7 @@ impl Session {
         START.call_once(|| {
             std::thread::spawn(|| smol::run(futures::future::pending::<()>()));
         });
-        session.set_blocking(false);
+        session.set_blocking(true);
```

Comment:
This makes the whole session synchronous so it basically takes away the point of this library. Am I missing something?

Reply:
It was working, but not asynchronously:)
Fixed https://github.com/spebern/async-ssh2/issues/14, but awfully.

I've implemented better solution in https://github.com/0xdeafbeef/async-ssh2/blob/master/src/session.rs